src/main/java/org/apache/maven/plugins/war/AbstractWarMojo.java

```diff
@@ -557,12 +563,26 @@ public void buildWebapp( MavenProject mavenProject, File webapplicationDirectory
 
         }
 
+        if ( !isCopyWebResources() && getWebResources() != null )
+        {
+            for ( Resource resource : getWebResources() )
+            {
+                if ( resource.isFiltering() )
+                {
+                    throw new MojoExecutionException(
```

Comment:
Plugin does filtering on copying resources. If we disable resources coping and provide them archiver directly from their directory then we have to do filtering by ourself.  As we don't need filtering it was not implemented.
Here we simply letting know that this use case is not supported.